test/IntegrationTests/MSTest.Acceptance.IntegrationTests/GenericTestMethodTests.cs

```diff
@@ -22,59 +22,59 @@ public async Task TestDifferentGenericMethodTestCases()
             """
             failed AMethodWithBadConstraints \(0\) \(\d+ms\)
               GenericArguments\[0], 'System\.Int32', on 'Void AMethodWithBadConstraints\[T]\(T\)' violates the constraint of type 'T'\.
-                at .+?
+                .+?
             failed NonParameterizedTestMethod \(\d+ms\)
               The generic test method 'NonParameterizedTestMethod' doesn't have arguments, so the generic parameter cannot be inferred\.
-                at .+?
+                .+?
             failed ParameterizedMethodSimple \(1\) \(\d+ms\)
               Assert\.Fail failed\. Test method 'ParameterizedMethodSimple' did run with parameter '1' and type 'System\.Byte'\.
-                at .+?
+                .+?
             failed ParameterizedMethodSimple \(2\) \(\d+ms\)
               Assert\.Fail failed\. Test method 'ParameterizedMethodSimple' did run with parameter '2' and type 'System\.Int32'\.
-                at .+?
+                .+?
             failed ParameterizedMethodSimple \("Hello world"\) \(\d+ms\)
               Assert\.Fail failed\. Test method 'ParameterizedMethodSimple' did run with parameter 'Hello world' and type 'System\.String'\.
-                at .+?
+                .+?
```

Comment:
Some of these are becoming `--- End of stack trace from previous location ---`

The stack trace was previously not so helpful anyways, so I think the change is fine.